### PR TITLE
SNOW-160668 add wss to NodeJS Driver

### PR DESCRIPTION
**Pull request by sfc-gh-stakeda**
Add WhiteSource scan script to NodeJS driver.

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -13,11 +13,13 @@ on:
...
     build:
         name: Build
-        runs-on: ubuntu-16.04
+        runs-on: ubuntu-latest
         steps:
             - uses: actions/checkout@v1
             - name: Build
               shell: bash
+              env:
+                WHITESOURCE_API_KEY: ${{ secrets.WHITESOURCE_API_KEY }}
```
Comment (sfc-gh-stakeda) [inline .github/workflows/build-test.yml:22]: A new secret maintained by Github.

File: ci/container/build_component.sh [hunk 2]
```diff
@@ -10,13 +10,14 @@ THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
...
 npm pack
-npm install --package-lock-only
-# The internal repository 10.180.20.84 doesn't support audit command.
-# We no longer use this repo and need to find out it the new version does support audit.
+npm install
 rm -f ~/.npmrc
 npm audit
 
+[[ -n "$WHITESOURCE_API_KEY" ]] && $THIS_DIR/wss.sh
+
```
Comment (sfc-gh-stakeda) [inline ci/container/build_component.sh:20]: This kicks off wss.sh

File: ci/container/wss.sh [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+rm -f wss-unified-agent.jar 
+curl -LO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
+
+SCAN_CONFIG=wss-nodejs-agent.config
+cat > $SCAN_CONFIG <<CONFIG
+###############################################################
+# WhiteSource Unified-Agent configuration file
+###############################################################
+# NPM SCAN MODE: package.json (or package-lock.json)
```
Comment (sfc-gh-stakeda) [inline ci/container/wss.sh:26]: This is a copied template as is.

File: ci/image/Dockerfile.nodejs-centos6-default-build [hunk 4]
```diff
@@ -56,6 +61,9 @@ RUN chmod +x /usr/local/bin/gosu
 COPY scripts/entrypoint.sh /usr/local/bin/entrypoint.sh
 RUN chmod +x /usr/local/bin/entrypoint.sh
 
+# Java
+RUN yum -y install java-1.8.0-openjdk-devel
+
```
Comment (sfc-gh-stakeda) [inline ci/image/Dockerfile.nodejs-centos6-default-build:66]: The build container needs Java to run WSS.

**Comment by sfc-gh-jfan** [comment]
hey, @sfc-gh-stakeda  I am able to confirm the whitesource stuff is correctly configured. For the container or other stuffs, you might need another peer reviewer I guess?

**Comment by sfc-gh-stakeda** [comment]
@sfc-gh-mkeller can you take a quick look?

**Pull request merged**